internal/app/machined/pkg/controllers/network/operator/dhcp6.go

```diff
@@ -245,7 +245,12 @@ func (d *DHCP6) renew(ctx context.Context) (time.Duration, error) {
 
 	d.parseReply(reply)
 
-	return reply.Options.OneIANA().Options.OneAddress().ValidLifetime, nil
+	options := reply.Options.OneIANA()
```

Comment:
@smira let's try adding tests around these functions that might return `nil`

Reply:
I wish I knew how to do that... we can mock responses, but that would be too much mocking?